src/components/organisms/HelpCenter/HelpCenter.tsx

```diff
@@ -25,18 +27,32 @@ export const HelpCenter: React.FC = () => {
     [startLoading]
   );
 
+  const handleBackClicked = () => {
+    setUrl("");
+  };
+
   return (
     <>
       {url ? (
         <>
           {isLoading && <LoadingPage />}
-          <iframe
-            className={"HelpCenter__iframe"}
-            onLoad={stopLoading}
-            frameBorder="0"
-            src={url}
-            title="Help Center"
-          />
+          <div className="HelpCenter__iframeContainer">
+            {!isLoading && (
+              <Button
+                customClass="HelpCenter__backButton"
+                onClick={handleBackClicked}
+              >
+                Back
+              </Button>
```

Comment:
`ButtonNG`? Has support for loading animation as well.

Reply:
@goran-peoski-work fixed in d544e65e1fe2d1354769ff619730270cb4d4b5d7